src/extension/background-script/actions/blocklist/delete.ts

```diff
@@ -0,0 +1,20 @@
+import db from "../../db";
+
+const deleteBlocklist = async (
+  message: { args: { host: string } },
```

Comment:
For all these messages types should be used, like we do here:
- https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/actions/accounts/delete.ts#L4
- https://github.com/getAlby/lightning-browser-extension/blob/master/src/types.ts#L87